x-pack/plugins/canvas/public/lib/run_interpreter.ts

```diff
@@ -17,10 +17,11 @@ interface Options {
  */
 export async function interpretAst(
   ast: ExpressionAstExpression,
-  variables: Record<string, any>
+  variables: Record<string, any>,
+  input: any
```

Comment:
nit: may want to default this to null for backwards compatibility

```suggestion
  input: any = null
```